File: javascript-sdk/src/common/enums.ts
export const enum CustomHeaders {
  ApiKey = "X-Api-Key",
  Timestamp = "X-Timestamp",
  Nonce = "X-Nonce",
  Signature = "X-Signature",
}

export const enum QueryParseMode {
  Repeat, // https://api.example.com?q1=a&q1=b&q2=c
  Bracket, // https://api.example.com?q1[]=a&q1[]=b&q2=c
  Comma, // https://api.example.com?q1=a,b&q2=c
}

File: dart-sdk/lib/src/common/enums.dart
enum CustomHeaders {
  apiKey(value: 'X-Api-Key'),
  timestamp(value: 'X-Timestamp'),
  nonce(value: 'X-Nonce'),
  signature(value: 'X-Signature');

  const CustomHeaders({required this.value});

  final String value;
}
